playbook-website/app/javascript/site_styles/_visual_guidelines.scss

```diff
@@ -35,15 +41,41 @@
       }
     }
 
-    .two { margin-left: 50px; }
-    .three { margin-left: 100px; }
-    .four { margin-left: 150px; }
-    .five { margin-left: 200px; }
-    .six { margin-left: 250px; }
-    .seven { margin-left: 300px; }
-    .eight { margin-left: 350px; }
-    .nine { margin-left: 400px; }
-    .ten { margin-left: 450px; }
+    .two {
+      margin-left: 50px;
+    }
+
+    .three {
+      margin-left: 100px;
+    }
+
+    .four {
+      margin-left: 150px;
+    }
+
+    .five {
+      margin-left: 200px;
+    }
+
+    .six {
+      margin-left: 250px;
+    }
+
+    .seven {
+      margin-left: 300px;
+    }
+
+    .eight {
+      margin-left: 350px;
+    }
+
+    .nine {
+      margin-left: 400px;
+    }
+
+    .ten {
+      margin-left: 450px;
+    }
```

Comment:
```suggestion
```

Unless you are utilizing this it has been removed from master

Reply:
@cvargaspower This all needs to match what is on Master. You just need to commit Jason's suggestion. I think you only need really the one line of code in this file on line 3: `@import "playbook-ui/dist/tokens/opacity";`